.ci/generate_bear_requirements.py

```diff
@@ -44,6 +47,10 @@
 
 PROJECT_BEAR_DIR = os.path.abspath(os.path.join(PROJECT_DIR, 'bears'))
 
+SUPPORTED_INSTANCES = ('PipRequirement',
```

Comment:
move these to the next line indented, as we will want to add entries at the end, so also use a trailing comma with closing bracket on the following line.